### PR TITLE
[TOOLS-4334] [CMT] enhancements to accommodate multi schema

### PR DESCRIPTION
**Pull request by win777c**
* Please, Refer to [TOOLS-4334](http://jira.cubrid.org/browse/TOOLS-4334)

### CODE REVIEW

File: com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/editor/controller/MigrationProgressUIController.java [hunk 1]
```diff
@@ -450,20 +450,42 @@ public void run(IProgressMonitor monitor) throws InvocationTargetException,
...
+			}
+		}
+		return new String[] {};
+	}
+
+	public String[] updateTableExpData(String owner, String tableName, long exp) {
+		if (exp <= 0) {
+			return new String[] {};
+		}
+		for (String[] item : tableItems) {
```
Comment (airnet73) [inline com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/editor/controller/MigrationProgressUIController.java:463]: If only check item[0] and item[5] , I don't think i  need "for loop".

File: com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/editor/controller/MigrationProgressUIController.java [hunk 2]
```diff
@@ -472,23 +494,41 @@ public void run(IProgressMonitor monitor) throws InvocationTargetException,
 	 * 
 	 * @return row data updated
 	 */
+	public String[] updateTableImpData(String owner, String tableName, long imp) {
+		
+		for (String[] item : tableItems) {
```
Comment (airnet73) [inline com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/editor/controller/MigrationProgressUIController.java:499]: If only check item[0] and item[5] , I don't think i  need "for loop".

File: com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/editor/controller/MigrationProgressUIController.java [hunk 3]
```diff
@@ -472,23 +494,41 @@ public void run(IProgressMonitor monitor) throws InvocationTargetException,
...
+			}
+			if (item[0].equals(tableName) && item[5].equalsIgnoreCase(owner)) {
+				return getItemForImpData(imp, item);
+			}
+		}
+		return new String[] {};
+	}
+	
 	public String[] updateTableImpData(String tableName, long imp) {
 		for (String[] item : tableItems) {
```
Comment (airnet73) [inline com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/editor/controller/MigrationProgressUIController.java:512]: If only check item[0] , I don't think i  need "for loop".

File: com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/editor/controller/MigrationProgressUIController.java (hunk 1 above)
Comment (win777c) [reply com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/editor/controller/MigrationProgressUIController.java:463]: * It is not the end by one time comparison, It needs to compare to columns (Table name, Table schema) in all items (Table Items) to display and update the status during UI processing. So, I think the for loop statement is required.

File: com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/editor/controller/MigrationProgressUIController.java (hunk 2 above)
Comment (win777c) [reply com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/editor/controller/MigrationProgressUIController.java:499]: * I have the same opinion on this as I wrote above.

File: com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/editor/controller/MigrationProgressUIController.java (hunk 3 above)
Comment (win777c) [reply com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/editor/controller/MigrationProgressUIController.java:512]: * I have the same opinion on this as I wrote above.

**Review by airnet73 — APPROVED** [review]

**Pull request merged**